_data/companies.yml

```
@@ -1434,11 +1434,11 @@
   last_update: 2020-03-04
 
 - name: Slido
-  wfh: Encouraged
+  wfh: Restricted
```

Could you explain what restricted means in this sense?

---

Jeez, nice catch, that's a typo. Thx!

```suggestion
  wfh: Required
```